test/3_givenProvider-ganache.js

```diff
@@ -0,0 +1,29 @@
+var assert = require('assert');
+var ganache = require('ganache-cli');
+var Web3 = require('../packages/web3');
+var Basic = require('./sources/Basic');
+
+describe('web.providers.givenProvider (ganache)', function(){
+    var web3;
+    var accounts;
+    var basic;
+
+    var basicOptions = {
+        data: Basic.bytecode,
+        gasPrice: '1',
+        gas: 4000000
+    };
+
+    // TODO: This triggers Nodes max listeners warning.
```

Comment:
It would be great if an issue would exist for this because otherwise will we forget about.

Reply:
This is currently tracked at #1648